scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -36,21 +36,24 @@ workflow GvsCreateVATfromVDS {
     }
 
     parameter_meta {
-        ancestry_file: {
-            help: "TSV file in GCS where the first column is the research ID and the last is the derived ancestry"
+        project_id: {
+            help: "Google project ID for the GVS BigQuery dataset"
         }
         dataset_name: {
             help: "BigQuery dataset name for GVS"
         }
+        ancestry_file: {
+            help: "TSV file in GCS where the first column is the research ID and the last is the derived ancestry"
+        }
         filter_set_name: {
             help: "name of the filter set used to generate the callset in GVS"
         }
+        sites_only_vcf: {
+            help: "Optional sites-only VCF file. If defined, generation of a sites-only VCF from the VDS will be skipped"
+        }
         output_path: {
             help: "GCS location (with a trailing '/') to put temporary and output files for the VAT pipeline"
         }
-        project_id: {
-            help: "Google project ID for the GVS BigQuery dataset"
-        }
         vds_path: {
```

Comment:
might be useful to add that if `sites_only_vcf` is passed, the WDL will error if `vds_path` is set